Microsoft/helium
# Foundation for CosmosDB telemetry (App Insights)

Inserted starter code for AppInsights telemetry into CosmosDB provider.

To test locally:
1.  Setup or get access to an Azure infrastructure for testing, and set environment variables and KeyVault secrets for needed resources (see https://github.com/Microsoft/helium/blob/master/docs/azure-infrastructure.md)
2.  Clone from branch aka/cosmosdb-instrumentation
3.  `npm install`
4.  `npm run build`
5.  `npm start`
6.  Use a browser or Postman to send requests to the API server (`localhost:3000/api/movies` or `localhost:3000/api/movies/<movie id>` for example)
7.  After 3-5 minutes, check your instance of Application Insights (Investigate->Search) for telemetry data that was sent from your queries.
